src/legacy/core_plugins/kibana/public/dashboard/saved_dashboard/saved_dashboard_references.test.ts

```diff
@@ -38,6 +39,7 @@ describe('extractReferences', () => {
           },
         ]),
       },
+      references: [],
```

Comment:
why is this needed?

Reply:
yes! :) TypeScript was sad without this!